examples/api/src/app/api/open-graph/lib/url-handlers/local-fetch.ts

```diff
@@ -5,7 +5,8 @@ import { chainById } from "../chains/chain-index";
 import { fetchNFTMetadata } from "../util";
 
 async function localFetchHandler(url: string): Promise<UrlMetadata> {
-  const userAgent = "bot";
+  const userAgent =
```

Comment:
this is considered an impolite bot (improperly identifying oneself) - we should probably add a comment here why we're doing this. We should probably also add "ModBot" to the end of the string to identify ourselves

Reply:
Got it, fixed